CHANGELOG.rst

```diff
@@ -1,5 +1,8 @@
 restructuredtext-lint changelog
 ===============================
+
```

Comment:
Please delete this leading newline, it breaks the existing convention for our CHANGELOG.